.github/workflows/master.yml

```diff
@@ -95,9 +95,10 @@ jobs:
         sudo apt-get install -y libcurl4-openssl-dev
         sudo R CMD javareconf
     - name: Install dependencies
+      working-directory: mlflow/R/mlflow
       run: |
         install.packages("devtools")
-        remotes::install_deps('mlflow/R/mlflow', dependencies = TRUE, upgrade = FALSE)
+        remotes::install_deps('.', dependencies = TRUE, upgrade = FALSE)
```

Comment:
Run `remotes::install_deps` in `mlflow/R/mlflow` to allow R to find `.Rprofile`.

Reply:
where is .Rprofile file ?

Reply:
oh, in user home directory